nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -282,6 +311,45 @@ def quantize_output(self, output, wrapper, **kwargs):
         out = self._dequantize(module, out)
         return out
 
+    def export_model(self, model_path, calibration_path=None):
+        """
+        Export quantized model weights and calibration parameters(optional)
+
+        Parameters
+        ----------
+        model_path : str
+            path to save quantized model weight
+        calibration_path : str
+            (optional) path to save quantize parameters after calibration
+        """
+        assert model_path is not None, 'model_path must be specified'
+        self._unwrap_model()
+        calibration_config = {}
+        support_op = [torch.nn.Conv2d, torch.nn.Linear, torch.nn.ReLU6]
```

Comment:
why `ReLU6`?

Reply:
Because activation layers also need to be set bit width in inference framework like **tensorrt**, we have to choose activation layers which we can support during calibration. I have tested `ReLU6` in some examples and it can be fully supported by **tensorrt**. But I am not sure whether other activation ops can be supported.

Reply:
what types of ops does QAT support in our current implementation?

Reply:
Removed `support_op` constrain in `export_model()`.